### PR TITLE
us placetype local and more

### PR DESCRIPTION
**Pull request by nvkelso**
Will fix a portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/2154. Sets placetype local, names, populations, and statoids properties for CARTO

### CODE REVIEW

File: data/856/326/49/85632649.geojson [hunk 1]
```diff
@@ -287,6 +290,7 @@
     "src:lbl_centroid":"mapshaper",
     "src:population":"wd",
     "src:population_date":"2019",
+    "src:population_year":"2019",
```
Comment (stepps00) [inline data/856/326/49/85632649.geojson:293]: Just catching this.. but I think this should be `src:population_date`, no? I don't see a `src:population_year` property https://github.com/whosonfirst/whosonfirst-properties/tree/main/properties/src

**Comment by stepps00** [comment]
Noticed an unexpected `src:` property.. but other than that, great additions.

File: data/856/326/49/85632649.geojson (hunk 1 above)
Comment (nvkelso) [reply data/856/326/49/85632649.geojson:293]: Great catch, thank you! I'll fix... Affects 340 records.

Comment (nvkelso) [reply data/856/326/49/85632649.geojson]: Resolved via https://github.com/whosonfirst-data/whosonfirst-data-admin-us/pull/152/commits/1fb9a46222b48de5f455655cfda3399a9afae6c8 in this repo, and other commits in other repos...

**Pull request merged**